CodeConverter/VB/MethodBodyExecutableStatementVisitor.cs

```diff
@@ -66,8 +66,19 @@ public override SyntaxList<StatementSyntax> VisitLocalDeclarationStatement(CSS.L
         private StatementSyntax ConvertSingleExpression(CSS.ExpressionSyntax node)
         {
             var exprNode = node.Accept(_nodesVisitor);
-            if (!(exprNode is StatementSyntax))
-                exprNode = SyntaxFactory.ExpressionStatement((ExpressionSyntax)exprNode);
+            var expression = exprNode as ExpressionSyntax;
+            if (expression != null) {
```

Comment:
Nit: For cases like this, try to use the more modern syntax to be consistent with the rest of the codebase:
```csharp
if (exprNode is ExpressionSyntax expression)
```

Reply:
Unfortunately, modern syntax is acceptable only for boolean variables.